packages/database/src/runPostMigration.js

```diff
@@ -5,24 +5,22 @@
 
 import { arrayToDbString } from './utilities';
 
-const EXCLUDED_TABLES_FROM_TRIGGER_CREATION = [
-  'api_request_log',
-  'dhis_sync_log',
-  'error_log',
-  'migrations',
-  'feed_item',
-  'userSession',
-  'spatial_ref_sys', // Reference table provided by postgis
-  'legacy_report',
-  'report',
-  'ancestor_descendant_relation',
-  'psss_session',
-  'lesmis_session',
-  'admin_panel_session',
-  'tupaia_web_session',
-  'analytics',
-  'data_service_sync_group', // config is too large for triggers
-  'data_table', // config is too large for triggers
+const TABLES_REQUIRING_TRIGGER_CREATION = [
+  'entity_hierarchy',
+  'analytics_refresher',
+  'survey_response',
+  'entity',
+  'country',
+  'clinic',
+  'geographical_area',
+  'option',
+  'option_set',
+  'permission_group',
+  'question',
+  'survey',
+  'survey_group',
+  'survey_screen',
+  'survey_screen_component',
```

Comment:
I did a bit of a dig and found these tables all needed triggers:
```suggestion
  'access_request',
  'ancestor_descendant_relation',
  'answer',
  'clinic',
  'country',
  'data_element',
  'entity',
  'entity_hierarchy',
  'entity_relation',
  'geographical_area',
  'indicator',
  'map_overlay',
  'map_overlay_group',
  'meditrak_device',
  'option',
  'option_set',
  'permission_group',
  'question',
  'survey',
  'survey_group',
  'survey_response',
  'survey_screen',
  'survey_screen_component',
  'user_entity_permission',
```
I think the main ones missing were due to the `ChangeHandler` classes (eg. `AnalyticsRefresher` and `EntityHierarchyCacher`) and the model notifiers